vision/google/cloud/vision/entity.py

```diff
@@ -61,12 +61,12 @@ def from_api_repr(cls, response):
         :returns: Instance of ``EntityAnnotation``.
         """
         bounds = Bounds.from_api_repr(response.get('boundingPoly'))
-        description = response['description']
-        locale = response.get('locale', None)
+        description = response.get('description')
```

Comment:
Are **all** of these necessary? Like are there **any** keys guaranteed to be around?